packages/web-runtime/src/extensionPoints.ts

```diff
@@ -0,0 +1,14 @@
+import { CustomComponentExtension, ExtensionPoint } from '@ownclouders/web-pkg'
+import { computed } from 'vue'
+
+export const topBarCenterExtensionPoint: ExtensionPoint<CustomComponentExtension> = {
+  id: 'app.runtime.header.center',
+  extensionType: 'customComponent',
+  multiple: true
+}
+
+export const extensionPoints = () => {
+  return computed<ExtensionPoint<any>[]>(() => {
```

Comment:
```suggestion
  return computed<ExtensionPoint<unknown>[]>(() => {
```

Reply:
`Vue: Type unknown does not satisfy the constraint Extension`

Reply:
then Extension? 😅

Reply:
Yep, works. Thanks 😅